clients/client-s3-control/src/commands/DeleteAccessPointCommand.ts

```diff
@@ -90,7 +90,7 @@ export class DeleteAccessPointCommand extends $Command
   .ep({
     ...commonParams,
     RequiresAccountId: { type: "staticContextParams", value: true },
-    AccessPointName: { type: "contextParams", name: "AccessPointName" },
+    AccessPointName: { type: "contextParams", name: "Name" },
```

Comment:
this doesn't seem to have any effect on the endpoint for a basic region/param combination, which may be why we didn't receive any reports or notice this

```js
import { DeleteAccessPointCommand, GetAccessPointCommand, S3ControlClient } from "@aws-sdk/client-s3-control";
import { getEndpointFromInstructions } from "@smithy/middleware-endpoint";

const client = new S3ControlClient({
  credentials: {
    accessKeyId: "redacted",
    secretAccessKey: "redacted",
  },
  region: "us-west-2",
});

{
  const endpoint = await getEndpointFromInstructions(
    { Name: "test", AccountId: "123" },
    DeleteAccessPointCommand,
    client.config
  );
  console.log("v3", endpoint.url.toString());
}

{
  const endpoint = await getEndpointFromInstructions(
    { Name: "test", AccountId: "123" },
    GetAccessPointCommand,
    client.config
  );
  console.log("v3", endpoint.url.toString());
}
```

output:
```
v3 https://123.s3-control.us-west-2.amazonaws.com/
v3 https://123.s3-control.us-west-2.amazonaws.com/
```

Reply:
The test I'd examined was https://github.com/aws/aws-sdk-js-v3/blob/03bb39fe253caad974494828ffeefddc9ffded78/codegen/sdk-codegen/aws-models/s3-control.json#L6094-L6132

Detailed logs in https://github.com/aws/aws-sdk-js-v3/pull/6373#issuecomment-2278753422